report_thesis/src/sections/problem_definition.tex

```diff
@@ -1,23 +1,60 @@
+
+The process of elemental quantification using \gls{libs} data is constrained by several factors:
+\begin{itemize}
+    \item \textbf{Data Dimensionality and Collinearity:} High dimensionality of spectral data coupled with multicollinearity, where multiple spectral features may exhibit strong correlations, complicates the modeling and analysis\cite{andersonImprovedAccuracyQuantitative2017}.
+    \item \textbf{Matrix Effects:} Used as a catch-all term that encompasses any effect that can cause the intensity of emission lines from an element to vary independent of that element's concentration. Different background materials can alter the emission intensities and pose significant challenges in accurately quantifying elemental concentrations. The spectra are complex due to the interaction of multiple physical processes including the coupling process between the laser photons and the target, self-absorption of optical emission lines within the plasma, recombination of elements into molecules, and collisional interactions in the plasma\cite{cleggRecalibrationMarsScience2017, andersonImprovedAccuracyQuantitative2017}.
+    \item \textbf{Data Availability:} Due to the high cost of data collection, datasets are often small, which may limit the generalizability of the models\cite{p9_paper}.
+\end{itemize}
+
+These constraints necessitate the development of advanced computational models that can effectively account for and mitigate these complexities to improve the accuracy and reliability of chemical composition analysis using \gls{libs} data.
+
+The input to our computational models consists of \gls{libs} spectral data, which includes intensity readings across a spectrum of wavelengths. This data is in the form of Clean, Calibrated Spectra\cite{andersonImprovedAccuracyQuantitative2017}, the output of level 1 processing as described by \citet{wiensPreflightCalibrationInitial2013}. The wavelength intensities are quantified in units of \\photon/shot/mm\textsuperscript{2}/sr/nm.
+
+We have:
 
-Subsequent missions, such as the \gls{mer} mission in 2003 and the \gls{msl} mission in 2012, sought to investigate whether Mars ever had the conditions to support life. 
-The Curiosity rover, part of the \gls{msl} mission, is equipped with the \gls{chemcam} instrument, which uses \gls{libs} to gather spectral data from geological samples on Mars~\cite{wiensChemcam2012}.
+\begin{itemize}
+    \item \textbf{Matrix $A_{t \times o}$}: This matrix denotes the chemical concentrations in weight percent for oxides $o$ across targets $t$.
```

Comment:
```suggestion
    \item \textbf{Matrix $A_{to}$}: This matrix denotes the chemical concentrations in weight percent for oxides $o$ across targets $t$.
```
I dont think the current subscript is correct. Source:
https://en.wikipedia.org/wiki/Matrix_(mathematics)

Reply:
Your source seems to agree?

![image](https://github.com/chhoumann/thesis-chemcam/assets/29108628/97dc6d83-8265-4009-bfae-d91bfbe666d5)

Reply:
Updated 
![image](https://github.com/chhoumann/thesis-chemcam/assets/29108628/52ea521f-2da5-42e4-852c-0a97d3f494b6)